doc/release_notes/release_notes_2.2.rst

```diff
@@ -40,25 +40,26 @@ Elkhart Lake and Tiger Lake processor support.
   <https://newsroom.intel.com/press-kits/intel-industrial-summit-2020>`_,
   Intel announced the latest additions to their
   enhanced-for-IoT Edge portfolio: the Intel® Atom® x6000E Series, Intel®
-  Pentium® and Intel® Celeron® N and J Series (all code named Elkhart Lake),
-  and 11th Gen Intel® Core™ processors (code named Tiger Lake-UP3). The ACRN
+  Pentium® and Intel® Celeron® N and J Series (all codenamed Elkhart Lake),
```

Comment:
Shouldn't we also be using the replacement for the reg, TM. etc. signed?

Reply:
Using either ® or |reg| works OK.  But you're right, I lean towards removing non-ASCII characters from the reST sources unless necessary.  I'll change them here.